### DISCUSSION TITLE
How could transform a string to DOM?

### DISCUSSION THREAD

**Discussion by thanhdatvo** [leptos-rs/leptos#1381] [Q&A] [answered]
How could transform a string to DOM?
Yew has this function but I don't know how to do this in Leptos
https://github.com/pvcresin/yew-markdown-preview/blob/ecc69668459d44b1066b18bfce0f677020458ae4/src/app.rs#L48

**Comment by gbj** [comment, marked as answer]
It looks to me like you want the `inner_html` method on `HtmlElement`, which you can also use as an attribute in the view macro:
`<div inner_html="<p>Hello!</p>"/>`

**Reply by thanhdatvo** [reply]
Brilliant!
Thanks!
